src/backend/templates/tracks.hbs

```diff
@@ -243,7 +243,7 @@
               {{/days}}
             </div>
           </div>
-          <div class="track-names">
+          <div class="col-md-9" >
```

Comment:
You can not remove the track names class 
@mahikaw correct way is  `<div class="track-names col-md-9">`